rpm/podman.spec

```diff
@@ -235,13 +235,20 @@ CGO_CFLAGS=$(echo $CGO_CFLAGS | sed 's/-flto=auto//g')
 CGO_CFLAGS=$(echo $CGO_CFLAGS | sed 's/-Wp,D_GLIBCXX_ASSERTIONS//g')
 CGO_CFLAGS=$(echo $CGO_CFLAGS | sed 's/-specs=\/usr\/lib\/rpm\/redhat\/redhat-annobin-cc1//g')
 
+%if %{defined copr_build}
+export BUILD_ORIGIN="%{?copr_username}/%{?copr_projectname}"
+%else
+export BUILD_ORIGIN="%{?dist_vendor} Official"
```

Comment:
What does this do for a manual `rpm -ba`? Does that say “Fedora Official”?

Reply:
Hmm yes. Though I see the current diff is broken on a local `rpmbuild -ba` anyway. I'll check if there's some special conditional for official build environments.

Reply:
looks like there's a _koji macro. Checking if it works..